corehq/apps/userreports/pillow.py

```diff
@@ -500,12 +500,12 @@ def _metrics_timer(self, step, config_id=None):
             timing_buckets=(.03, .1, .3, 1, 3, 10), tags=tags
         )
 
-    def _per_config_metrics_timer(self, step, config_id):
+    def _per_config_metrics_timer(self, step, config):
         tags = {
             'action': step,
         }
-        if settings.ENTERPRISE_MODE:
-            tags['config_id'] = config_id
+        tags['config_id'] = config._id
```

Comment:
Assuming many domains have more than one UCR config, this could be a pretty large set. Not totally against it, just something to keep an eye one.